noxfile.py

```diff
@@ -468,7 +468,12 @@ def test_jupyter_notebooks(session):
         session.skip(
             f"Not testing Jupyter notebook on Python {session.python}, supports [{','.join(versions)}]"
         )
-    session.install("jupyter", "nbval")
+    if "Windows" in get_current_os():
+        # pyzmq > 20.0.0 fails on Windows py3.8
+        # https://github.com/facebookresearch/hydra/pull/1348
+        session.install("jupyter", "nbval", "pyzmq==20.0.0")
+    else:
+        session.install("jupyter", "nbval")
```

Comment:
1. I have seen similar failures before. the release process for pyzmq is manual and they keep fucking it up.
2. When doing something like this, it's best to keep things simple: if 20.0.0 works just use it for everything.
3. They released a version today, I am guessing the version before it works. (https://pypi.org/project/pyzmq/#history)
4. There is already an issue reported [here](https://github.com/zeromq/pyzmq/issues/1496), it will probably be resolved by tomorrow.